dev/sg/sams/sg_sams.go

```diff
@@ -0,0 +1,122 @@
+// Package msp exports 'sg sams' commands for the Sourcegraph Accounts Management System.
+package sams
+
+import (
+	"encoding/json"
+
+	"github.com/urfave/cli/v2"
+	"golang.org/x/oauth2/clientcredentials"
+
+	sams "github.com/sourcegraph/sourcegraph-accounts-sdk-go"
+
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/category"
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/std"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+var clientCredentialsFlags = []cli.Flag{
+	&cli.StringFlag{
+		Name:    "sams-server",
+		Aliases: []string{"sams"},
+		EnvVars: []string{"SG_SAMS_SERVER_URL"},
+		Value:   "https://accounts.sgdev.org",
+		Usage:   "URL of the Sourcegraph Accounts Management System (SAMS) server",
+	},
+	&cli.StringFlag{
+		Name:     "client-id",
+		EnvVars:  []string{"SG_SAMS_CLIENT_ID"},
+		Usage:    "Client ID of the Sourcegraph Accounts Management System (SAMS) client",
+		Required: true,
+	},
+	&cli.StringFlag{
+		Name:     "client-secret",
+		EnvVars:  []string{"SG_SAMS_CLIENT_SECRET"},
+		Usage:    "Client secret for the Sourcegraph Accounts Management System (SAMS) client",
+		Required: true,
+	},
+	&cli.StringSliceFlag{
+		Name:    "scopes",
+		Aliases: []string{"s"},
+		EnvVars: []string{"SAMS_SCOPES"},
```

Comment:
```suggestion
		EnvVars: []string{"SG_SAMS_SCOPES"},
```

Unless there is a reason not to be consistent? (e.g. a package is expecting `SAMS_SCOPES `?)

Reply:
I think it's better to have this be listed explicitly actually (i.e. not allowed to be set via env), as copy-pasting it into a PR test plan with `-s` is clearer, and this isn't sensitive

Reply:
```suggestion
```